addon/components/truncate-multiline.js

```diff
@@ -193,6 +194,38 @@ export default Ember.Component.extend(ResizeHandlerMixin, {
    */
   resizeOnInsert: false,
 
+  /**
+   * Helper function to determine if an attribute has changed.
+   * @private
+   * @param attrs {Object}
+   * @param name {String}
+   * @return {Boolean}
+   */
+  _didAttrChange(attrs, name) {
+    let oldAttr = attrs.oldAttrs[name];
+    let newAttr = attrs.newAttrs[name];
+
+    if (typeof oldAttr === "undefined" || typeof newAttr === "undefined") {
+      return false;
```

Comment:
This needs to check for when one is undefined and the other is not. The attribute _did_ change in that case.